transports/transports.go

```diff
@@ -22,6 +23,7 @@ func init() {
 	for _, t := range []types.ImageTransport{
 		directory.Transport,
 		docker.Transport,
+		tarfile.Transport,
```

Comment:
Consider adding a smoke test for the naming to `transports_test.go`. Not strictly necessary.